SimCore/src/SimCore/DetectorConstruction.cxx

```diff
@@ -77,7 +77,7 @@ static bool isInTargetRegion(G4LogicalVolume* vol,
 }
 
 /**
- * isInTargetRegion
+ * isInTargetOnly
```

Comment:
You should do `git restore SimCore/src/SimCore/DetectorConstruction.cxx` since you didnt mean to touch this file

Reply:
If the `VolumeTests` are used elsewhere, they should be removed from this file so that they aren't duplicated. If the `VolumeTests` are not used elsewhere, you can `git restore` this file and `git rm` the VolumeTests header and source files.

Reply:
The volume tests are used elsewhere so they should be removed from this file and the `VolumeTests.h` header included so that they can also be used when assigning biasing operators to volumes.